### ISSUE TITLE
Documentation colors hurt my eyes

### ISSUE THREAD

**Issue by quickfur** [s-ludwig/std_data_json#33]
On this page: http://s-ludwig.github.io/std_data_json/stdx/data/json/parser.html
the example code block contains black text on a dark grey background, which makes it basically unreadable on my screen / to my eyes. Could we pretty please use higher color contrast here, so that the docs are actually readable?

**Comment by s-ludwig** [comment]
Which browser/OS is that on? It should be white text on dark gray, and medium gray comments.

**Comment by quickfur** [comment]
Firefox/Linux.  In Firebug, it shows that the `<pre>` block did not inherit any `color` settings that may have been set on ancestor elements, or perhaps `color` was never specified in the stylesheet?

**Comment by quickfur** [comment]
P.S. Yeah, I don't see `color` defined anywhere in the ancestors of the `<pre>` block. Are you by any chance relying on the browser's default value for `color`? Because that can be unreliable across browsers, or even different versions / OS versions of the same browser.

**Comment by quickfur** [comment]
Oh, I see what the problem is.  I have Javascript disabled by default, and apparently the Javascript is responsible for adding syntax highlighting to the code block.  I guess you can close this issue if you want.  Would be nice if the CSS specified a default color for code blocks without depending on Javascript, though.

**Comment by s-ludwig** [comment]
Ah okay, I'll test this tomorrow, but probably this will be fixed by regenerating the docs. The current ddox version uses libdparse to perform the syntax highlighting statically.

**Comment by quickfur** [comment]
Thanks!
